es74x/build.gradle

```diff
@@ -52,11 +52,11 @@ compileScala {
 
 dependencies {
     compile project(path: ':core')
-
-    compile 'org.elasticsearch:elasticsearch:' + esVersion
-    compile 'org.elasticsearch.plugin:transport-netty4-client:' + esVersion
-
-    compile group: 'org.apache.logging.log4j',              name: 'log4j-core',           version: '2.11.0'
+    compile             group: 'org.elasticsearch',                 name: 'elasticsearch' ,             version: esVersion
+    compileClasspath    group: 'org.codelibs.elasticsearch.lib',    name: 'plugin-classloader',         version: esVersion
+    compileClasspath    group: 'org.locationtech.spatial4j',        name: 'spatial4j',                  version: '0.7'
```

Comment:
it this only need for 74x? Could you double check it, because I've recently tried your method and I didn't need this dependency (at least for ROR startup and basic ES queries)

Reply:
Yes. I did 
1. `./gradlew clean compileTestScala`,
2.  `refresh all Gradle projects` in intellij, 
3. `rebuild project`, run `WholeEs`,
 and failed with `ClassNotFoundException`.